cranelift/codegen/meta/src/shared/settings.rs

```diff
@@ -39,7 +39,7 @@ pub(crate) fn define() -> SettingGroup {
             representation, performs various rewrites, and then converts it back. This can result in
             better optimization, but at the cost of a longer compile time.
         "#,
-        false,
+        true,
```

Comment:
Does this mean it will do egraph optimizations even if opt_level is set to none?

Reply:
This is a work-in-progress branch and I'm doing some hacking on some benchmarking infrastructure, and I wanted to name points-in-time by hashes only, without a separate config. We will definitely not enable opts if opts are disabled. Please note the commit comment and please disregard throwaway commits on my work-in-progress branch; there will be ample opportunity to review when the real PRs come.

Reply:
Didn't see the commit message as I was looking at the github diff view for all changes since the last notification.